About the package version downgrade in the Volo.Abp.EntityFrameworkCore.MySQL.csproj file

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Description

The project was upgraded from version 8.x to 9.1.0, and a compilation error occurred.

> error NU1605: Detected package downgrade:  Microsoft.EntityFrameworkCore.Relational from 9.0.0 to 9.0.0-preview.7.24405.3. Reference the package directly from the project to select a different version

I noticed the following reference in the Volo.Abp.EntityFrameworkCore.MySQL.csproj file.
https://github.com/abpframework/abp/blob/3616e843cfdee671317941fe5b478c1c8bd180df/framework/src/Volo.Abp.EntityFrameworkCore.MySQL/Volo.Abp.EntityFrameworkCore.MySQL.csproj#L25
Is this line in the file redundant?

### Reproduction Steps

_No response_

### Expected behavior

_No response_

### Actual behavior

_No response_

### Regression?

_No response_

### Known Workarounds

_No response_

### Version

9.1.0

### User Interface

Common (Default)

### Database Provider

EF Core (Default)

### Tiered or separate authentication server

None (Default)

### Operation System

Windows (Default)

### Other information

_No response_

--- a/framework/src/Volo.Abp.EntityFrameworkCore.MySQL/Volo.Abp.EntityFrameworkCore.MySQL.csproj
+++ b/framework/src/Volo.Abp.EntityFrameworkCore.MySQL/Volo.Abp.EntityFrameworkCore.MySQL.csproj
@@ -22,7 +22,6 @@
 
   <ItemGroup>
     <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" VersionOverride="9.0.0-preview.7.24405.3" />
   </ItemGroup>
 
 </Project>

